src/app/main-page-content/main-page-content.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="profile-content">
+  <div class="menu">
+    <app-search-input></app-search-input>
+    <app-add-wish-modal></app-add-wish-modal>
+  </div>
+  <mat-tab-group mat-align-tabs="center">
+    <mat-tab label="My wishes">
+      <div *ngIf="this.wishService.wishes.length > 0 && this.wishService.filteredWishes[0] !== null; else empty">
```

Comment:
Suggestion:
<div *ngIf="this.wishService.wishes.length && this.wishService.filteredWishes[0] !== null; else empty">